spec/factories/schools.rb

```diff
@@ -3,7 +3,7 @@
 FactoryBot.define do
   factory :school do
     urn { Faker::Number.unique.number(digits: 6) }
-    name { Faker::Educator.secondary_school }
```

Comment:
https://github.com/faker-ruby/faker/blob/46130d9212932c5ad3e5523413ef2189937f55af/lib/locales/en/educator.yml#L4-L20

The pool is limited